### PR TITLE
Tests: add collectibles tests

### LINKED ISSUE #3261: Add automation for collectibles
To increase automation coverage, the following test were added to collectibles suite:

- Verify Send button is disabled for disconnected wallet
- Verify Send button is disabled for non-owner
- Verify that when 2 NFTs are selected, actions and tx details are correct in Review step
- Verify that when 1 NFTs is selected, there is no Actions block in Review step

### PR DESCRIPTION
**Pull request by mike10ca**
## What it solves

Resolves #3261 

## How this PR fixes it
To increase automation coverage, the following test were added to collectibles suite:

- Verify Send button is disabled for disconnected wallet
- Verify Send button is disabled for non-owner
- Verify that when 2 NFTs are selected, actions and tx details are correct in Review step
- Verify that when 1 NFTs is selected, there is no Actions block in Review step

## How to test it

- Run Cypress tests and observe outcome.

### CODE REVIEW

File: src/components/tx/DecodedTx/index.tsx [hunk 1]
```diff
@@ -78,13 +78,17 @@ const DecodedTx = ({
       )}
 
       <Accordion elevation={0} onChange={onChangeExpand} sx={!tx ? { pointerEvents: 'none' } : undefined}>
-        <AccordionSummary expandIcon={<ExpandMoreIcon />} className={accordionCss.accordion}>
+        <AccordionSummary
+          data-testid="accordion-summary"
```
Comment (katspaugh) [inline src/components/tx/DecodedTx/index.tsx:82]: I would name it something more specific to decoded txs as we use the same AccordionSummary in many other components.

File: src/components/tx/DecodedTx/index.tsx [hunk 2]
```diff
@@ -78,13 +78,17 @@ const DecodedTx = ({
...
+          expandIcon={<ExpandMoreIcon />}
+          className={accordionCss.accordion}
+        >
           <span style={{ flex: 1 }}>Transaction details</span>
 
           {decodedData ? decodedData.method : tx?.data.operation === OperationType.DelegateCall ? 'Delegate call' : ''}
         </AccordionSummary>
 
-        <AccordionDetails>
+        <AccordionDetails data-testid="accordion-details">
```
Comment (katspaugh) [inline src/components/tx/DecodedTx/index.tsx:91]: Same here.

File: src/components/tx/DecodedTx/index.tsx (hunk 1 above)
Comment (mike10ca) [reply src/components/tx/DecodedTx/index.tsx]: Done

File: src/components/tx/DecodedTx/index.tsx (hunk 2 above)
Comment (mike10ca) [reply src/components/tx/DecodedTx/index.tsx]: Done

**Review by katspaugh — APPROVED** [review]

**Review by francovenica — APPROVED** [review]
LGTM

**Pull request merged**